### DISCUSSION TITLE
Authorization: User is not permitted to create this Twitter Circle tweet. (459)

### DISCUSSION THREAD

**Discussion by tsngu** [dimdenGD/OldTwitter#595] [Q&A] [answered]
Hello, I do not know if I'm at the right place for it, but I downloaded the extension on Firefox and I still can't post in circle ?

The error is : Authorization: User is not permitted to create this Twitter Circle tweet. (459)

Thank you

**Comment by dimdenGD** [comment, marked as answer]
You can't post to circles anymore
